client/settings/fraud-protection/index.js

```diff
@@ -0,0 +1,501 @@
+import TipIcon from 'wcpay/icons';
+import './style.scss';
+
+const ExceedsDollarAmountRuleUSDStore = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<li>
+				{ __( 'An order exceeds ', 'woocommerce-payments' ) }{ ' ' }
+				<strong>{ __( '$1,000.00.', 'woocommerce-payments' ) }</strong>
+			</li>
+		);
+	}
+
+	return (
+		<li>
+			{ __( 'An order exceeds', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '$1,000.00', 'woocommerce-payments' ) } </strong>
+			{ __( 'or', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '10 items.', 'woocommerce-payments' ) }</strong>
+		</li>
+	);
+};
+
+const ExceedsDollarAmountRuleNotUSDStore = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<li>
+				{ __(
+					'An order exceeds the equivalent of',
+					'woocommerce-payments'
+				) }{ ' ' }
+				<strong>
+					{ __( '$1,000.00 USD', 'woocommerce-payments' ) }
+				</strong>{ ' ' }
+				{ sprintf(
+					__( 'in %s.', 'woocommerce-payments' ),
+					props.defaultCurrency
+				) }
+			</li>
+		);
+	}
+
+	return (
+		<li>
+			{ __(
+				'An order exceeds the equivalent of',
+				'woocommerce-payments'
+			) }{ ' ' }
+			<strong>{ __( '$1,000.00 USD', 'woocommerce-payments' ) } </strong>{ ' ' }
+			{ sprintf(
+				__( 'in %s', 'woocommerce-payments' ),
+				props.defaultCurrency
+			) }{ ' ' }
+			{ __( 'or', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '10 items.', 'woocommerce-payments' ) }</strong>
+		</li>
+	);
+};
+
+const ProtectionLevelModalNotice = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<Notice
+				className="component-notice--is-info"
+				status="info"
+				isDismissible={ false }
+			>
+				<div className="component-notice__content--flex">
+					<TipIcon className="component-notice__icon" />
+					<p>
+						{ __(
+							'Offers the highest level of filtering for stores, but may catch some legitimate transactions',
+							'woocommerce-payments'
+						) }
+					</p>
+				</div>
+			</Notice>
+		);
+	}
+
+	return (
+		<Notice
+			className="component-notice--is-info"
+			status="info"
+			isDismissible={ false }
+		>
+			<div className="component-notice__content--flex">
+				<TipIcon className="component-notice__icon" />
+				<p>
+					{ __(
+						"Provides a standard level of filtering that's suitable for most business.",
+						'woocommerce-payments'
+					) }
+				</p>
+			</div>
+		</Notice>
+	);
+};
+
+const ProtectionLevels = () => {
+	const [ isStandardModalOpen, setStandardModalOpen ] = useState( false );
+	const [ isHighModalOpen, setHighModalOpen ] = useState( false );
+	const { currencies } = useCurrencies();
+	const storeCurrency = currencies.default ? currencies.default : {};
+	const isDefaultCurrencyUSD = 'USD' === storeCurrency.code;
+
+	const [
+		currentProtectionLevel,
+		updateProtectionLevel,
+	] = useCurrentProtectionLevel();
+
+	const handleLevelChange = ( level ) => {
+		updateProtectionLevel( level );
+	};
+
+	return (
+		<>
```

Comment:
The Fragment `<>` can be removed as `<fieldset>` is the only direct child of it.

Reply:
Thanks for catching that. Removed in [bc73d05](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/bc73d05e0a294a833d3644dc36932f949f31a250).